dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -16,10 +16,10 @@ antlr4-runtime/4.13.1//antlr4-runtime-4.13.1.jar
 aopalliance-repackaged/3.0.3//aopalliance-repackaged-3.0.3.jar
 arpack/3.0.3//arpack-3.0.3.jar
 arpack_combined_all/0.1//arpack_combined_all-0.1.jar
-arrow-format/15.0.0//arrow-format-15.0.0.jar
-arrow-memory-core/15.0.0//arrow-memory-core-15.0.0.jar
-arrow-memory-netty/15.0.0//arrow-memory-netty-15.0.0.jar
-arrow-vector/15.0.0//arrow-vector-15.0.0.jar
+arrow-format/15.0.2//arrow-format-15.0.2.jar
+arrow-memory-core/15.0.2//arrow-memory-core-15.0.2.jar
+arrow-memory-netty/15.0.2//arrow-memory-netty-15.0.2.jar
+arrow-vector/15.0.2//arrow-vector-15.0.2.jar
```

Comment:
To @panbingkun , we had better handle `arrow` dependency first independently.

Reply:
Okay, let me separate them.  Just a moment.

Reply:
Done